src/main/java/com/powsybl/openloadflow/ac/DistributedSlackOnLoadOuterLoop.java

```diff
@@ -63,24 +65,30 @@ public double run(List<ParticipatingElement<LfBus>> participatingElements, int i
             LfBus bus = participatingBus.element;
             double factor = participatingBus.factor;
 
-            double targetP = bus.getLoadTargetP();
-            double newTargetP = targetP - remainingMismatch * factor;
+            double loadTargetP = bus.getLoadTargetP();
+            double newLoadTargetP = loadTargetP - remainingMismatch * factor;
 
             // We stop when the load produces power.
-            if (newTargetP <= 0) {
-                newTargetP = 0;
+            if (newLoadTargetP <= 0) {
+                newLoadTargetP = 0;
                 loadsAtMin++;
                 it.remove();
             }
 
-            if (newTargetP != targetP) {
-                if (LOGGER.isTraceEnabled()) {
-                    LOGGER.trace("Rescale '{}' active power target: {} -> {}",
-                            bus.getId(), targetP * PerUnit.SB, newTargetP * PerUnit.SB);
+            if (newLoadTargetP != loadTargetP) {
+                LOGGER.trace("Rescale '{}' active power target: {} -> {}",
+                        bus.getId(), loadTargetP * PerUnit.SB, newLoadTargetP * PerUnit.SB);
+                if (powerFactorConstant) { // https://github.com/powsybl/powsybl-open-loadflow/issues/110
```

Comment:
I think a sentence to explain the purpose of this block of code would be more useful and convenient than a link to the issue lists.